bash.bashrc

```diff
@@ -0,0 +1,8 @@
+alias ls='ls --color=auto'
+alias o='grep --color=auto'
+alias dashb='cd /ceph/src/pybind/mgr/dashboard_v2'
```

Comment:
I think these changes are too user-specific to be included in the default image. This Docker container might be used for other things than Dashboard v2 development.